src/planner/binder/statement/bind_copy.cpp

```diff
@@ -125,7 +126,8 @@ BoundStatement Binder::BindCopyTo(CopyStatement &stmt) {
 	if (is_remote_file) {
 		use_tmp_file = false;
 	} else {
-		bool is_file_and_exists = config.file_system->FileExists(stmt.info->file_path);
+		bool is_file_and_exists =
+		    config.file_system->FileExists(stmt.info->file_path, ClientData::Get(context).file_opener.get());
```

Comment:
Can this use `FileSystem::GetFileSystem(context)` instead? That way we don't need to pass a FileOpener explicitly. It's unclear to me why this is using `config.file_system` in the first place.

Reply:
Ok will make the change here :)